song-server/pom.xml

```diff
@@ -160,6 +160,11 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.vavr</groupId>
```

Comment:
where is this used ?

Reply:
AnalysisServiceImpl.java

Reply:
https://github.com/overture-stack/SONG/pull/743/files#diff-3ad7aede999b78dca2c87366270896b4684688e893d4ba6168d1951ad06d02c3R232